app/services/openid_connect_service.rb

```diff
@@ -0,0 +1,12 @@
+class OpenidConnectService
```

Comment:
Good point!

I don't see this class as generating the private key, just parsing it. But good point, I'll move it and clean up the other code that reads it as well.